tests/validation/exec_test.go

```diff
@@ -0,0 +1,33 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rancher/rio/tests/testutil"
+)
+
+func execTests(t *testing.T, when spec.G, it spec.S) {
+
+	var service testutil.TestService
+	var otherService testutil.TestService
+
+	when("multiple services are running", func() {
+		it.Before(func() {
+			service.Create(t, "ibuildthecloud/demo:v1")
+			otherService.Create(t, "ibuildthecloud/demo:v3")
+		})
+
+		it.After(func() {
+			service.Remove()
+			otherService.Remove()
+		})
+
+		it("should allow accessing services directly from within other service pods in the same namespace", func() {
+			//rio exec -it d-test@v1 /bin/sh -c "apt-get update && apt-get install -y --fix-missing curl && curl --version"
```

Comment:
?

Reply:
Oh I can remove that comment. That's just the command it runs